src/app/side-bar/dashboard/dashboard.component.scss

```diff
@@ -0,0 +1,30 @@
+@import "src/assets/styles/variables";
+
+.dashboard-container {
+  width: auto;
+  &__graph {
+    margin: 0 16px;
+    background-color: $floral-white;
+  }
+  &__card-title {
+      padding: 0 20px;
+    }
+  &__card-img {
+    margin-left: 24px;
+    position: relative;
+    height: 256px;
+    &-caption {
+      position: absolute;
+      width: 93%;
+      bottom: 0;
+      padding: 12px 0 12px 4px;
+      color: white;
```

Comment:
use variable

Reply:
resolved .......................Thank you